config/config.go

```diff
@@ -18,6 +18,10 @@ const (
 	DefaultDymintDir      = ".dymint"
 	DefaultConfigDirName  = "config"
 	DefaultConfigFileName = "dymint.toml"
+	MinBlockTime          = 200 * time.Millisecond
+	MaxBlockTime          = 6 * time.Second
+	MaxBatchSubmitMaxTime = 1 * time.Hour
+	MaxBlockSkewSupported = 432000 // equivalent to 24h blocks at max block rate
```

Comment:
seems way too much imo. 

what's the euristic for choosing this? how long will it take to recover from this? (i.e be in sync again with settlement, i.e submission rate to cover 432000.)